test/distributed/_composable/test_fully_shard.py

```diff
@@ -464,6 +465,16 @@ def test_state_dict_root_fully_shard(self):
         composable_sd = composable_module.state_dict()
         self._check_state_dict_parity(local_sd, composable_sd)
 
+        # Validate load
+        load_composable = fully_shard(
+            copy.deepcopy(local_model),
+            policy=ModuleWrapPolicy({UnitModule})
+        )
+        _zero_model(load_composable)
```

Comment:
Should we check that `load_composable` after the `_zero_model()` call has parameters different from the model that generated `composable_sd` to make sure this `load_state_dict()` was not a no-op? (For example, if somehow due to a `fully_shard` bug, `_zero_model()` did not actually zero the model, or if somehow even the original model was already zeroed, then this test passes even if `load_state_dict` does nothing.)

Reply:
good point!